SkyWars/src/vixikhd/skywars/arena/Arena.php

```diff
@@ -83,6 +83,11 @@ class Arena implements Listener {
 
     /** @var Level $level */
     public $level = null;
+    
+    // 5 Teams for plugin
+    public $reds = [];
```

Comment:
Look to my EggWars plugin how to make teams easier.... Having array for each team doesn't look good